ExampleMod/Content/NPCs/ExamplePerson.cs

```diff
@@ -347,12 +347,12 @@ public class ExamplePersonProfile : ITownNPCProfile
 
 		public Asset<Texture2D> GetTextureNPCShouldUse(NPC npc) {
 			if (npc.IsABestiaryIconDummy && !npc.ForcePartyHatOn)
-				return ModContent.Request<Texture2D>("ExampleMod/Content/NPCs/ExamplePerson");
+				return ModContent.Request<Texture2D>("ExampleMod/Content/NPCs/ExamplePerson", AssetRequestMode.ImmediateLoad);
 
 			if (npc.altTexture == 1)
-				return ModContent.Request<Texture2D>("ExampleMod/Content/NPCs/ExamplePerson_Party");
+				return ModContent.Request<Texture2D>("ExampleMod/Content/NPCs/ExamplePerson_Party", AssetRequestMode.ImmediateLoad);
 
-			return ModContent.Request<Texture2D>("ExampleMod/Content/NPCs/ExamplePerson");
+			return ModContent.Request<Texture2D>("ExampleMod/Content/NPCs/ExamplePerson", AssetRequestMode.ImmediateLoad);
```

Comment:
ImmediateLoad shouldn't be anything required or expected. Use `asset.Wait?.Invoke()` calls instead.